theano/tensor/nnet/nnet.py

```diff
@@ -384,6 +384,7 @@ def c_code(self, node, name, inp, out, sub):
             }
         }
 
+        #pragma omp parallel for schedule(static, 32)
```

Comment:
Hi nouiz,

Could you please help review it again? :) I have made softmax and softmaxGrad inherit from gof.OpenMPOp and commit the code.

Many thanks.